src/safety_report_generator.py

```diff
@@ -60,13 +61,15 @@ def insert_vulnerabilites_into_report(self, scanned_vulnerabilities):
                 "advisory": advisory,
                 "spec": spec,
                 "reason_to_ignore": "N/A",
+                "ignored": False,
```

Comment:
This seems like a good idea.

Can we add ` vulnerability_details ["ignored"]=True` after this line as well: https://github.com/aws/deep-learning-containers/pull/3465/files#diff-97f48d676eb69a20887ba357107023d932b65fa54a67240edc030b610d2835dbR170

Reply:
Done